.github/workflows/googlejavaformat-action.yml

```diff
@@ -1,16 +1,33 @@
 name: Format
 
-on:
-  push:
-    branches-ignore:
-      - master
+on: [push, pull_request]
 
 jobs:
 
   formatting:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2 # v2 minimum required
-      - uses: axel-op/googlejavaformat-action@v3
+      - uses: LGoodDatePicker/googlejavaformat-action@v3
```

Comment:
The `@v3` already freezes the action to the released version `v3`. 
My concerns were not about the author changing the action but the action code itself.  As seen [here](https://github.com/LGoodDatePicker/googlejavaformat-action/blob/d655be08f03f518babebbaab6145c084845fb024/index.js#L141):
```javascript
let release = await getRelease(releaseId)
core.debug(`release is ${typeof release}`);
const assets = release['assets'];
core.debug(`assets is ${typeof assets}`);
const downloadUrl = assets.find(asset => asset['name'].endsWith('all-deps.jar'))['browser_download_url'];
core.info(`Downloading executable to ${executable}`);
await curl(downloadUrl, `-o ${executable}`)
await executeGJF(['--version']);
```
The action uses a quite complex logic do determine which *GoogleJavaFormat* version to use and then downloads and executes it. 
My actual concern is the *GoogleJavaFormat* executable that is downloaded and used might have an issue in the future. 
My solution was to exclude the master branch from the format action to avoid any broken commits on the master branch in that scenario.

I know I'm an unusually risk averse person so if that scenario is of no concern to you we can just keep formatting all branches.